routing/routing_tests/turns_generator_test.cpp

```diff
@@ -4,8 +4,9 @@
 #include "routing/turns.hpp"
 #include "routing/turns_generator.hpp"
 
-#include "geometry/mercator.hpp"
+#include "indexer/ftypes_matcher.hpp"
```

Comment:
Because ftypes::HighwayClass::Trunk defined in ftypes_matcher.hpp is used in this file below.